lms/djangoapps/grades/admin/tests/utils.py

```diff
@@ -0,0 +1,21 @@
+"""
+Utilities for tests involving the feature flags
+controlling persistent grading.
+"""
+from lms.djangoapps.grades.admin.models import CoursePersistentGradesFlag, PersistentGradesEnabledFlag
+
+
+def set_persistent_grades_feature_flags(
+        global_flag,
+        override_course_settings=False,
+        course_id=None,
+        course_setting=False
+):
```

Comment:
Nit: for readability, can we rename the parameters to something like:
override_course_settings  -> enabled_for_all_courses
course_setting -> enabled_for_course